spec/models/tslr_claim_spec.rb

```diff
@@ -291,4 +298,46 @@
       end
     end
   end
+
+  describe "#complete!" do
+    context "when claim is valid" do
```

Comment:
Here's where the "valid" wording used in the trait ends up being potentially misleading/confusing to the reader.